test/integration/requests/ledger.ts

```diff
@@ -0,0 +1,45 @@
+import { assert } from 'chai'
+import _ from 'lodash'
+
+import { LedgerRequest, LedgerResponse } from 'xrpl-local'
+
+import serverUrl from '../serverUrl'
+import { setupClient, suiteClientSetup, teardownClient } from '../setup'
+import { verifySuccessfulResponse } from '../utils'
+
+// how long before each test case times out
+const TIMEOUT = 20000
+
+describe('ledger', function () {
+  this.timeout(TIMEOUT)
+
+  before(suiteClientSetup)
+  beforeEach(_.partial(setupClient, serverUrl))
+  afterEach(teardownClient)
+
+  it('base', async function () {
+    const ledgerRequest: LedgerRequest = {
+      command: 'ledger',
+      ledger_index: 'validated',
+    }
+
+    const ledgerResponse: LedgerResponse = await this.client.request(
+      ledgerRequest,
+    )
+
+    const expectedResponse: LedgerResponse = {
+      id: ledgerResponse.id,
```

Comment:
You could create a helper function to check requests that does an `assert` compare that strips both objects of IDs, so that you don't need this part.

Reply:
I think this happens in `verifySubmittedTransaction`

Reply:
even better, just add it to `verifySuccessfulResponse`

Reply:
The main reason I'm doing this is because it allows for a very straightforward assert.deepEquals which ensures the shape matches, instead of going line by line with individual asserts. (Places where I borrow from the previous request are just an easier way of saying "I care that the info is there, but not what's in it") - Does that make sense? (Maybe I'm missing an antipattern that this is falling into?)